sqld/src/query.rs

```diff
@@ -130,6 +136,21 @@ pub enum Value {
     Blob(Vec<u8>),
 }
 
+impl Value {
+    /// Approx. of the size in bytes required to store the value as json
+    fn size_hint(&self) -> usize {
+        match self {
+            Value::Null => 4,
```

Comment:
Once we account for the type information, the additional cost of NULL is zero (NULL is encoded as `{"type":"null"}`).

Reply:
uhmm that's only true for hrana. Let me think of a better solution

Reply:
Do we still need to care about the legacy HTTP API? It even makes sense that both APIs apply the response size limit in the same way.